README.md

```diff
@@ -2432,6 +2432,37 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
```

Comment:
Before adapters, I think we should show how to turn off logging, `hazelcast.logging=off`. We would show how to set log levels in default logger but that is not available as of now. So we can show how to turn it off and continue.